src/sas/qtgui/UI/main_resources.qrc

```diff
@@ -4,6 +4,7 @@
     <file>res/right-grey.png</file>
     <file>res/file_send-128.png</file>
     <file>res/SendTo512.png</file>
+    <file>res/sasview-icon.ico</file>
     <file>res/ball.ico</file>
```

Comment:
> This is out-of-scope for this PR, but I think the work here highlights a standardization issue with the GUI. We should probably create a base window object that all widgets inherit from. This would standardize the look and feel of all elements.

Yeah, looking forward to the dark theme.

> What, if any, copyright and ownership issues do we need to consider for the new icon?

It's entirely original and I don't care what is done with it.

> You mention a colleague helping you with the design of the icon. Should the colleague be added to the contributor list?

Sure